src/gba/memory.c

```diff
@@ -383,6 +385,8 @@ static void GBASetActiveRegion(struct ARMCore* cpu, uint32_t address) {
 		LOAD_32(value, address & (SIZE_CART0 - 4), memory->rom); \
 	} else if (memory->mirroring && (address & memory->romMask) < memory->romSize) { \
 		LOAD_32(value, address & memory->romMask, memory->rom); \
+	} else if (memory->vfame.cartType && address < BASE_CART1) { \
```

Comment:
Come to think of it, does this not work with the cart mirrors, regardless of the WAITCNT setting? That seems kinda weird to me.